.github/workflows/ci.yaml

```
@@ -183,26 +183,31 @@ jobs:
+        uses: re-actors/alls-green@05ac9388f0aebcb5727afa17fcccfecd6f8ec5fe
+        with:
+          allowed-skips: >-
+            aur,
+            void,
+            winget,
+            lint,
+            tests,
+            android,
+            desktop,
+            docker
+          jobs: ${{ toJSON(needs }}
```

actionlint (1.7.8)

unexpected end of input while parsing arguments of function call. expecting ",", ")"

(expression)